src/conn/conn_api.c

```diff
@@ -3009,21 +3009,6 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
     WT_ERR(__wt_config_gets(session, cfg, "session_scratch_max", &cval));
     conn->session_scratch_max = (size_t)cval.val;
 
-    /*
-     * If buffer alignment is not configured, use zero unless direct I/O is also configured, in
-     * which case use the build-time default. The code to parse write through is also here because
-     * it is nearly identical to direct I/O.
-     */
-    WT_ERR(__wt_config_gets(session, cfg, "direct_io", &cval));
```

Comment:
I'm not sure what our policy is, but I'd be inclined to keep the configuration option in the public API, but deprecate it. Either return an error if it is passed in or silently ignore it (I'm not sure what the right choice is, probably the error?)

Reply:
Hi Alex, thanks for the review. 
I've put back the API's and left them undocumented. As discussed offline these configs will be silently ignored to be consistent with our previous API removals in WiredTiger. Changes are here [fe73c79](https://github.com/wiredtiger/wiredtiger/pull/10430/commits/fe73c7987eb312873344d3b3002f69e92fe688a5)